crux-metrics/src/crux/metrics/dropwizard/cloudwatch.clj

```diff
@@ -0,0 +1,36 @@
+(ns crux.metrics.dropwizard.cloudwatch
+  (:require crux.metrics.dropwizard)
+  (:import [io.github.azagniotov.metrics.reporter.cloudwatch CloudWatchReporter]
+           [java.io Closeable]
+           [software.amazon.awssdk.services.cloudwatch CloudWatchAsyncClient]
+           [software.amazon.awssdk.regions.providers DefaultAwsRegionProviderChain]
+           [software.amazon.awssdk.auth.credentials DefaultCredentialsProvider]
+           [java.util.concurrent TimeUnit]
+           [com.codahale.metrics MetricRegistry MetricFilter]))
+
+(defn ^CloudWatchReporter report
+  [^MetricRegistry reg {::keys [region dry-run? jvm-metrics?] :as args}]
+  (let [cwac (-> (CloudWatchAsyncClient/builder)
+                 (.region (.getRegion (new DefaultAwsRegionProviderChain)))
+                 (.credentialsProvider (DefaultCredentialsProvider/create))
+                 .build)]
+    ;; Might need seperate cwacs for regions
+    (-> (cond-> (CloudWatchReporter/forRegistry reg cwac "crux.metrics.dropwizard.cloudwatch")
+          jvm-metrics? .withJvmMetrics
+          dry-run? .withDryRun)
+        (.withGlobalDimensions (into-array String ["crux.node=metrics"]))
```

Comment:
what's this dimension doing?

Reply:
haven't a clue, added it to try and track it down in the metrics and never found it. I'll make it optional